src/NuGet.Core/NuGet.LibraryModel/AssemblyInfo.cs

```diff
@@ -3,5 +3,10 @@
 
 using System.Runtime.CompilerServices;
 
+#if SIGNED_BUILD
 [assembly: InternalsVisibleTo("NuGet.ProjectModel.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
 [assembly: InternalsVisibleTo("NuGet.Commands.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
+#else
+[assembly: InternalsVisibleTo("NuGet.ProjectModel.Test")]
```

Comment:
When is the build unsigned? 

We should be delay signing https://github.com/NuGet/NuGet.Client/blob/c5f496dae608298cb9d57768444717613400d3dc/build/config.props#L68-L71

Reply:
Yes, normally it is signing assemblies. 
But there is a property `SkipSigning` https://github.com/NuGet/NuGet.Client/blob/e88a5a03a1b26099f8be225d3ee3a897b2edb1d0/build/common.targets#L37-L39 which disables it.

Reply:
Are you setting that property manually or something else is? 
I'll approve the PR, just wanted to understand if/why it's set :) 

Can you please follow the steps here: https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md#contributing-step-by-step

* Please create a tracking issue.
* Please keep the PR template.

Reply:
I'm setting this property to disable signing, when I'm doing some local debugging of NuGet libraries and need to add extra dependencies to them. 
I don't want to bother with configuring the signing process for these extra dependencies so I simply disable it for all NuGet libraries as it is simpler.